src/style.css

```diff
@@ -29,18 +41,20 @@ header {
   display: flex;
 }
 
+p {
+  margin-left: 20px;
+}
+
 #left-column #leaderboard {
-  border: 2px solid black;
   margin-top: 30px;
   list-style-type: none;
 }
 
 .row {
   padding: 2px 0 2px 4px;
-}
-
-ul > li:nth-child(odd) {
-  background-color: rgb(217, 217, 217);
+  background-color: aliceblue;
+  margin: 10px 0 10px 0;
+  border-radius: 10px;
 }
 
 .refresh-btn {
```

Comment:
- [ ] As it is a requirement for this project, please make sure you keep the general layout of the wireframe, the `Refresh` and `Submit` buttons are right aligned in their various columns, please kindly fix that. See image below. 
***The wireframe***
![kar1](https://user-images.githubusercontent.com/65852621/141436131-0147e4f7-fcc7-4a56-9249-4afdd4490798.png)

***Your implementation***
![kar2](https://user-images.githubusercontent.com/65852621/141436219-0de81500-e8f6-434b-b65d-5ca33de4b6fb.png)